go.mod

```diff
@@ -119,3 +119,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/openshift/api => github.com/jhadvig/api v0.0.0-20240729130727-3ab071dc2aaa
```

Comment:
Should we put a hold on this PR until https://github.com/openshift/api/pull/1956 merges?